src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K,V>[] storage;
+    private int size;
+
+    public MyHashMap() {
+        storage = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Use the diamond operator to avoid unchecked assignment warning. It should be `storage = new Node<>[DEFAULT_INITIAL_CAPACITY];`